xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/xwiki.properties.vm

```diff
@@ -238,6 +238,83 @@ rendering.transformations = $xwikiRenderingTransformations
 #-#         \$sp and if you need a new line you'll need to use \$nl
 #-# rendering.macro.velocity.filter = indent
 
+#-------------------------------------------------------------------------------------
+# HTML Sanitization
+#-------------------------------------------------------------------------------------
+
+#-# [Since 13.10.7]
+#-# [Since 14.4.2]
+#-# [Since 14.5RC1]
+#-# The HTML sanitization strategy to use for user-generated content to avoid JavaScript injection. The following
+#-# strategies are available by default:
+#-# - secure (default): Only allows known elements and attributes that are considered safe. The following options
+#-#                     allow customizing its behavior.
+#-# - insecure:         Allows everything including JavaScript. Use this only if you absolutely trust everybody who can
+#-#                     write wiki syntax (in particular, all users, but also anonymous users commenting when enabled).
+# xml.htmlElementSanitizer = secure
+
+#-# [Since 13.10.7]
+#-# [Since 14.4.2]
+#-# [Since 14.5RC1]
+#-# Additional tags that should be allowed by the HTML sanitizer. These tags will be allowed in addition to the
+#-# already extensive built-in list of tags that are considered safe. Use with care to avoid introducing security
```

Comment:
Can you point out to the list of allowed tags?

Reply:
There are about 115 HTML, 60 SVG and 30 MathML tags that are allowed by default. I didn't want to duplicate this list here, in particular as it might change over time as new HTML features are added or problems with existing tags are discovered.

Reply:
>  didn't want to duplicate this list here

sure, but maybe you could provide an URL to the file containing the list?

Reply:
> sure, but maybe you could provide an URL to the file containing the list?

Do we have any standards how to link to the file? I could try linking to the file with a specific commit hash to ensure that the URL is stable, but then a) we would need to update this link whenever the list changes and b) I don't know the commit hash yet as the changes will probably be rebased again before they are merged, as I would like to have a clean and small commit history for easier cherry-picking.